src/test/java/Pages/HomePage.java

```diff
@@ -0,0 +1,23 @@
+package Pages;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+public class HomePage extends BasePage {
+
+    public HomePage(WebDriver givenDriver) {
+        super(givenDriver);
+    }
+
+    //Page Locators
+    By userAvatarIcon = By.cssSelector("img.avatar");
```

Comment:
We can make the xpath or css the class variable (Locator)

For example

![image](https://github.com/Test-Pro-Bootcamp/A49-QA-Automation-Framework/assets/104761482/d327eccd-5b31-455b-b586-1bbad7d539d5)